### PR TITLE
Add-Contact-Form-Home-page

### PR DESCRIPTION
**Pull request by moise10r**
In this section we implemented a basic UX : 
-> We used " Form" for getting information from the user with some validations.
-> We used  CSS flexbox to layout the content.

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -426,3 +426,118 @@ ul {
...
+  background-image: url("../images/header-shapes.png");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-top-left-radius: 5rem;
+  padding-bottom: 5rem;
+  position: relative;
+  opacity: 1;
+}
```
Comment (rloterh) [inline style/style.css:454]: - [ ] Your form design is great, and the layout color matches design specifications.  We however want to ensure that our design follows the design provided in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) template as much as possible. In the Figma design, we can see that the form elements are aligned to the middle (centered). The top left portion of the form looks quite rounded. Let's ensure to make this adjustment to make the UI comply with the design as much as possible.

**Review by rloterh — CHANGES_REQUESTED** [review]
# Changes Required! ♻️

Hi Moise 👋 👋, I'm Robert, your reviewer for this PR.

I must commend you for the nice work done👏.  You nailed almost all of the specifications of this project right. Kindly check the below comments for the necessary changes we can implement to make our application greater.

## Good points 👍
No Linters errors ✔️
Follow the readme template ✔️


## Aspects to improve ♻️

Check the comments under the review.

 On the readme file in the Authors section, you could fill it with your own information like (please, both of you):

Check the comment(s) under the review.

- On the ReadMe file in Author(s) section, you could fill it with your own information like:

## Author

👤 **Robert Loterh**

- GitHub: [@rloterh](https://github.com/rloterh)
- Twitter: [@RLoterh](https://twitter.com/RLoterh)
- LinkedIn: [LinkedIn](https://linkedin.com/in/robert-loterh)


Cheers and Happy coding!👏👏👏

Please feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

---
<em>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).</em>

File: style/style.css (hunk 1 above)
Comment (moise10r) [reply style/style.css:454]: Thank you a lot, I have already fixed it

**Review by Dipeshtwis — APPROVED** [review]
# Hi @moise10r , 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message and PR title, keep up the good work :muscle:
:+1: The project setup is well organized. Well Done.

**Please check my optional change and apply those before merging.** 

You have implemented all the suggestions from previous reviewer!  s**ince one very important feature missed in the previous review I am making this optional but I highly recommend to  apply in order to make your contact form really working.**

Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

-  Please integrate your form with formspree. By following these steps:
 -  Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). (Register here and use your own form_id)
- action value should be similar to https://formspree.io/f/{form_id}.
- Also use the same name attribute in each field required for formspree.

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

File: style/style.css (hunk 1 above)
Comment (Dipeshtwis) [reply style/style.css:454]: This issue was successfully addressed. :+1:

**Comment by Dipeshtwis** [comment]
Hi @moise10r,
You have mistaken me in the previous review. The form id should be your custom id which you'll get after registering into the formspree. Hope this makes it clear to you and feel free to correct the action part. Also, don't hesitate to ask here the question you arise while doing so or you can take help from your partner as well.

**Comment by rloterh** [comment]
**Hello Moise!**

I've just realized I missed one very important point, as pointed out by @Dipeshtwis, that <em>The form id should be your custom id which you'll get after registering into the formspree</em> is a very necessary implementation you needed to have improved in your form.

I'm sorry I couldn't point that out to you during my review. Kindly note, this was not on purpose 😊 

Happy coding! 🍷 💻 

Cheers!

**Pull request merged**